packages/theming/theme-types/src/Theme.types.ts

```diff
@@ -11,21 +11,31 @@ type SpacingValue = number | string;
 export interface Spacing {
   s2: string;
   s1: string;
+  m?: string;
```

Comment:
This shouldn't be optional anymore. We initially did this because both the old and new T-shirt size names had `m` as a possible value.

Reply:
Reverted the new addition.  
Also as mentioned in the description, we have to take the removal of spacing from theme as separate PR which should focus on entirely removal of spacing from theming.